docs/cluster_resources/node/README.md

```diff
@@ -0,0 +1,118 @@
+# Falcon Node Sensor
+
+## About Falcon Cloud Workload Protection
+
+Learn more at [product page](https://www.crowdstrike.com/cloud-security-products/falcon-cloud-workload-protection/) and [Linux sensor blog](https://www.crowdstrike.com/blog/tech-center/linux-protection/).
+
+## About FalconNodeSensor Custom Resource (CR)
+Falcon Operator introduces the FalconNodeSensor Custom Resource (CR) to the cluster. The resource is meant to install, configure, and uninstall the Falcon Linux Sensor on the cluster nodes. This resource deploys a kernel module to the Kubernetes nodes which runs as _privileged_.
+
+### FalconNodeSensor CR Configuration using CrowdStrike API Keys
+To start the FalconNodeSensor installation using CrowdStrike API Keys to allow the operator to determine your Falcon Customer ID (CID) as well as pull down the CrowdStrike Falcon Sensor container image, please create the following FalconNodeSensor resource to your cluster. You will need to provide CrowdStrike API Keys and CrowdStrike cloud region for the installation. It is recommended to establish new API credentials for the installation at https://falcon.crowdstrike.com/support/api-clients-and-keys, required permissions are:
+ * Falcon Images Download: Read
+ * Sensor Download: Read
+
+Example:
+```
+apiVersion: falcon.crowdstrike.com/v1alpha1
+kind: FalconNodeSensor
+metadata:
+  name: falcon-node-sensor
+spec:
+  falcon_api:
+    client_id: PLEASE_FILL_IN
+    client_secret: PLEASE_FILL_IN
+    cloud_region: autodiscover
+  node: {}
+  falcon: {}
+```
+
+### FalconNodeSensor CR Configuration with Falcon Customer ID (CID) and non-CrowdStrike Registry
+
+Example:
+```
+apiVersion: falcon.crowdstrike.com/v1alpha1
+kind: FalconNodeSensor
+metadata:
+  name: falcon-node-sensor
+spec:
+  falcon:
+    cid: PLEASE_FILL_IN
+  node:
+    image: myregistry/project/image:version
+```
+
+### FalconNodeSensor Reference Manual
+
+#### Falcon API Settings
+| Spec                                | Description                                                                                                                               |
+| :---------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------- |
+| falcon_api.client_id                | (optional) CrowdStrike API Client ID                                                                                                      |
+| falcon_api.client_secret            | (optional) CrowdStrike API Client Secret                                                                                                  |
+| falcon_api.client_region            | (optional) CrowdStrike cloud region (allowed values: autodiscover, us-1, us-2, eu-1, us-gov-1)                                            |
```

Comment:
Should this be `cloud_region`? Similar comment for `docs/cluster_resources/container/README.md`.

Reply:
Just saw this comment, I've crated #342 for this 🤙